test/CMakeLists.txt

```diff
@@ -1467,7 +1470,10 @@ if(iodaconv_bufr_query_ENABLED)
                     COMMAND bash
                     ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
                             netcdf
-                            "${Python3_EXECUTABLE} testinput/prepbufr_sfcshp_api.py"
+                            "${Python3_EXECUTABLE} ${PROJECT_SOURCE_DIR}/src/ncepbufr/prepbufr_sfcshp_api.py
+                            -d 202108010000
```

Comment:
pretty sure these dates are actually wrong so should replace with correct date

Reply:
1627768800

Reply:
that's in epoch ctime

Reply:
but the epoch is computed using the input time... chicken and egg there
think we just need to look at the input file and see if it doesn't have some information about which cycle it was valid for